src/distributions/semeru/installer.ts

```diff
@@ -182,21 +175,20 @@ export class SemeruDistribution extends JavaBase {
     }
 
     if (core.isDebug()) {
-      core.startGroup('Print information about available IBM Semeru versions');
-      console.timeEnd('semeru-retrieve-available-versions'); // eslint-disable-line no-console
+      core.startGroup('Print information about available versions');
+      console.timeEnd('Retrieving available versions for Semeru took'); // eslint-disable-line no-console
       core.debug(`Available versions: [${availableVersions.length}]`);
       core.debug(
         availableVersions.map(item => item.version_data.semver).join(', ')
       );
-      core.debug(JSON.stringify(availableVersions));
       core.endGroup();
     }
 
     return availableVersions;
   }
 
   private getPlatformOption(): string {
-    // Adoptium has own platform names so need to map them
+    // Adopt has own platform names so need to map them
```

Comment:
Eclipse Adoptium is the proper project name to use here

Reply:
Ok, I understand your reasoning now, though I don't think the platform names are any different between Adoptium and AdoptOpenJDK and anyone who knows about Adoptium (to the degree they know about its APIs :) ) is probably at least a little familiar with its heritage as AdoptOpenJDK. In any case, it's just a suggestion lobbed in from afar ;) .